tests/system/action/motion_state/test_create.py

```diff
@@ -23,6 +23,42 @@ def test_create(self) -> None:
         assert model.get("merge_amendment_into_final") == "undefined"
         assert model.get("css_class") == "lightblue"
 
+    def test_create_as_new_first_state(self) -> None:
+        self.set_models(
+            {
+                "meeting/1": {},
+                "motion_workflow/42": {
+                    "name": "test_name_fjwnq8d8tje8",
+                    "meeting_id": 1,
+                    "first_state_id": 1,
+                },
+                "motion_state/1": {
+                    "workflow_id": 42,
+                    "first_state_of_workflow_id": 42,
+                    "name": "first state one",
+                },
+            }
+        )
+        response = self.request_json(
+            [
+                {
+                    "action": "motion_state.create",
+                    "data": [
+                        {
+                            "name": "first state two",
+                            "workflow_id": 42,
+                            "first_state_of_workflow_id": 42,
```

Comment:
Also add a test with
```
"workflow_id": 42,
"first_state_of_workflow_id": 43,
```
With both workflows existing. This must fail, too.

Reply:
done